types/wallet.d.ts

```diff
@@ -12,6 +15,12 @@ export default interface Wallet {
     update(): Promise<void>;
     broadcast(tx: tapyrus.Transaction): Promise<string>;
     balance(colorId?: string): Promise<Balance>;
+    utxos(colorId?: string): Promise<Utxo[]>;
+    estimatedFee(tx: tapyrus.Transaction): number;
+    transfer(params: TransferParams[], changePubkeyScript: Buffer): Promise<{
```

Comment:
Isn't it better for transfer not only to build a remittance transaction but also to sign and broadcast it?

Reply:
I'll try to fix.

Reply:
Yes. Updated at 2bb76eb